### PR TITLE
[USH-489] Images accessibility 1 of 5A: mat-Icons, Images and SVGs in the sidebar

### PR DESCRIPTION
**Pull request by MugoBrian**
@Ifycode . As per your request.

[[USH-489] Images accessibility 1 of 5A: mat-Icons, Images and SVGs in the sidebar](https://linear.app/ushahidi/issue/USH-489/images-accessibility-1-of-5a-mat-icons-images-and-svgs-in-the-sidebar)

### CODE REVIEW

**Comment by MugoBrian** [comment]
[[USH-489] Images accessibility 1 of 5A: mat-Icons, Images and SVGs in the sidebar](https://linear.app/ushahidi/issue/USH-489/images-accessibility-1-of-5a-mat-icons-images-and-svgs-in-the-sidebar)

**Comment by tuxpiper** [comment]
@MugoBrian please update the branch with the latest changes from development (using the "Rebase branch" method above)

Is this work in progress or ready to be reviewed?  If the review is stuck, please let me know.

**Comment by MugoBrian** [comment]
@tuxpiper it is ready to be reviewed. @Ifycode is reviewing it.

File: apps/web-mzima-client/src/app/shared/components/company-info/company-info.component.html [hunk 1]
```diff
@@ -7,11 +7,16 @@
 >
   <mat-expansion-panel-header>
     <mat-panel-title class="company-info__header">
-      <img [src]="logo" [alt]="title" class="company-info__logo" *ngIf="logo" />
+      <!-- Adding Logo at the end (thought it was more descriptive) -->
```
Comment (Ifycode) [inline apps/web-mzima-client/src/app/shared/components/company-info/company-info.component.html:10]: Let's start with the comments. Remove this comment @MugoBrian

File: apps/web-mzima-client/src/app/shared/components/company-info/company-info.component.html [hunk 2]
```diff
@@ -7,11 +7,16 @@
...
+      <img [src]="logo" [alt]="title + 'Logo'" class="company-info__logo" *ngIf="logo" />
       <span class="company-info__title">{{ title }}</span>
-      <mat-icon svgIcon="arrow-down" class="company-info__header__arrow"></mat-icon>
+      <!-- The arrow down icon is an interactive icon -->
```
Comment (Ifycode) [inline apps/web-mzima-client/src/app/shared/components/company-info/company-info.component.html:13]: Remove this comment @MugoBrian

File: apps/web-mzima-client/src/app/shared/components/submit-post-button/submit-post-button.component.html [hunk 3]
```diff
@@ -5,5 +5,7 @@
   [iconOnly]="!(isDesktop$ | async)!"
 >
   <span *ngIf="isDesktop$ | async">{{ 'app.add_new_post' | translate }}</span>
-  <mat-icon icon svgIcon="plus"></mat-icon>
+  <!-- Added aria-hidden true (I noticed all mat-icons in the side nav the aria-hidden is set to true in default 
+    (or perhaps there's a configuration I missed)) -->
```
Comment (Ifycode) [inline apps/web-mzima-client/src/app/shared/components/submit-post-button/submit-post-button.component.html:9]: Remove this comment @MugoBrian

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: apps/web-mzima-client/src/app/shared/components/submit-post-button/submit-post-button.component.html [hunk 4]
```diff
@@ -5,5 +5,7 @@
...
+  <mat-icon icon svgIcon="plus" aria-hidden="true"></mat-icon>
```
Comment (Ifycode) [inline apps/web-mzima-client/src/app/shared/components/submit-post-button/submit-post-button.component.html:10]: You can safely remove `aria-hidden=true` as miroslav said in the previous PR. It is already added by default.

File: apps/web-mzima-client/src/app/shared/components/sidebar/sidebar.component.html [hunk 5]
```diff
@@ -3,7 +3,7 @@
     <span class="sidebar__logo" *ngIf="isDesktop">
       <img
         src="../../../../assets/images/ushahidi-logo.svg"
-        alt="ushahidi"
+        alt="Ushahidi Company Platform Logo"
         width="172"
```
Comment (Ifycode) [inline apps/web-mzima-client/src/app/shared/components/sidebar/sidebar.component.html:7]: "Ushahidi Logo" is fine as alt text.

**Review by Ifycode — CHANGES_REQUESTED** [review]

**Comment by Ifycode** [comment]
In summary (about your comments) @MugoBrian.
Instead of writing your explanations inside the code in form of an explanation to the reviewer, you can always just click on the  "File changes" tab here on the pull request. Then click on the green "review changes" button. Then you can add such very explanatory comments through that medium, so that you are able to submit cleaner code.

File: apps/web-mzima-client/src/app/shared/components/submit-post-button/submit-post-button.component.html (hunk 4 above)
Comment (MugoBrian) [reply apps/web-mzima-client/src/app/shared/components/submit-post-button/submit-post-button.component.html]: All Done!

**Comment by Ifycode** [comment]
Closing this one because we will have to restructure first.

**Pull request closed without merging**